README.md

```diff
@@ -1,99 +1,122 @@
-Court Case Service
-==================
-Service to access court cases imported from HMCTS Libra court lists
+# Court Case Service
+### Service to access court cases imported from HMCTS Libra court lists
+
 
-Dev Setup
 ---
 
-In order to run the service locally, a postgres database and some other backend services are required, the easiest way to run locally is using the [docker-compose.yml](docker-compose.yml) file which will pull down the latest version.
 
-```docker-compose up```
+## Setup your machine
 
-In order for the authentication to function between services locally you must add the following line to your `/etc/hosts` file before starting:
+In the root folder of your machine start by checking if HomeBrew is installed on your computer with the command : `brew -version` 
```

Comment:
The bit about Java 11 is definitely useful but the rest might be a bit more detail than we'd normally go into. It's useful stuff but different users might have different ways of installing dependencies (for example Windows/Linux users) and might encounter different problems so trying to document for every possibility quickly becomes intractable!  

Happy to leave it in as it as we don't have huge amounts of documentation, you're right we've been a bit lax on that so I'm very glad you're taking the time to make improvements!

Reply:
How about making a section for Mac and leave an open door for people Window/Linux? Maybe the next after me will be happy to fill these section :) ?

Reply:
I'd leave it as it is, we only use macs anyway so it was mostly to illustrate a point, if some intrepid Windows user does come along and then they can of course submit a PR!